hack/tools/protoc-gen-jsonschema/module/enum.go

```diff
@@ -36,10 +37,15 @@ func (m *Module) schemaForEnum(enum pgs.Enum, rules *validate.EnumRules) (jsonsc
 
 		if len(rules.NotIn) > 0 {
 			schemas = append(schemas, jsonschema.Not(m.schemaForEnumIn(enum, rules.NotIn)))
+			required = true
```

Comment:
I don't think `not_in` generally implies `required` - the logic I've been applying for `required` is "does the rule exclude the zero value". Arguably we could loop over the `rules.NotIn` and set `required = true` if one of the values is zero - we should probably similarly do that for the `in` case so that `required = false` if zero is included.

Reply:
Good point.